README.md

```diff
@@ -3,90 +3,121 @@ GOV.UK Frontend ·
 
-Guidance for building a GOV.UK service - [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/).
+## Before using GOV.UK Frontend
 
-Want to help us build GOV.UK Frontend? See [contributing](#contributing)
+You should start by familiarising yourself with the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/), which contains guidance and examples of components, styles and patterns you can use to design  government platforms and services. 
+
+## How to use GOV.UK Frontend 
+
+There are 2 ways to use GOV.UK Frontend:
+* we recommend using <a href="#option1">option 1</a>, which requires using node package manager (NPM) 
+* you can also use <a href="#option2">option 2</a>, which requires you to download the assets (CSS, JavaScript) from GitHub
+
+<a id="option1"></a>
+
+# Option 1: Use NPM
 
 ## Requirements
+To use GOV.UK Frontend with NPM you must:
+
+1. Install the long-term support (LTS) version of [Node.js](https://nodejs.org/en/), which includes NPM.
+2. Create a [package.json file](https://docs.npmjs.com/files/package.json) if you don’t already have one. You can create a default package.json file by running `npm init` from the root of your application.
+
+## Installation
+GOV.UK Frontend is currently in private beta. You will need to log in to NPM using credentials provided by the Design System team.
+
+Run the following command to log in:
+`npm login`
+
+Enter the username, password and email address you were provided with, when prompted. 
+
+You can install all components or one or more individual components depending on your needs.
+
+To install all components, run:
+`npm install --save @govuk-frontend/all`
+
+
+To install individual components (for example, a button), run:
+`npm install --save @govuk-frontend/button`
+
+You can find a list of all components in the [packages directory](https://github.com/alphagov/govuk-frontend/tree/master/packages). Each component has a README with installation and usage instructions.
+
+After you have installed GOV.UK Frontend the @govuk-frontend package will appear in your node_modules folder.
+
+## Import assets
 
-1) [NodeJS](https://nodejs.org/en/) installed
-2) [package.json](https://docs.npmjs.com/files/package.json) file
+Import styles into your main Sass file.
 
-This file is needed to install a GOV.UK Frontend NPM package in your application. Most often it resides in the root of your application. You can create this file interactively using a `npm init` command.
+To import all components, add the line:
+`@import "@govuk-frontend/all/all";`
 
+To import an individual component (for example, a button), add the line:
+`@import "@govuk-frontend/button/button";`
 
-## Install all components
 
-```
-npm install --save @govuk-frontend/all
-```
+To resolve your @import declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [load paths](url here) in Ruby.
```

Comment:
`(url here)` for Ruby load paths needs completing

Reply:
Alex looking into this

Reply:
Updated.